packages/kotti-ui/source/kotti-translation/locales/de-DE.ts

```diff
@@ -24,6 +25,14 @@ export const deDE: KottiTranslation.Messages = {
 			IS_EMPTY: 'ist leer',
 		},
 		clearAllLabel: 'Alles Löschen',
+		currency: {
+			EQUAL: 'gleich',
```

Comment:
i think you can just follow what sonarjs was yelling aboout, and define a constant particularly for the number operations
> better maintainance in the future - if at any point we need to change one of those strings, we change one line instead of 3 and potentially even missing one of them